yaksh/templates/yaksh/course_detail.html

```diff
@@ -141,61 +141,47 @@
                         <thead>
                             <tr class="yakshlight">
                             <th>Sr No.</th>
-                            <th>Students</th>
-                            <th>Total</th>
+                            <th>Email</th>
```

Comment:
Below for Email column if email does not exists then name is displayed.
I would suggest that name and email to be separate columns.

Reply:
A User/Teaching assistant recommended that the Name or Email is not *as* import as the Roll number of the students.

I'll be including the Roll Number and keeping the rest of the structure unchanged as more columns will mean a crowded UI.